doc/user-guide.md

```diff
@@ -124,12 +122,12 @@ type Storage interface {
 }
 ```
 
-**Note:** For a real use case, you may decide to keep the `.proto` file and the generated `.pb.go` files in a separate directory/package, and import that package (the generated Gorums API) into to your main application.
+**Note:** For a real use case, you may decide to keep the `.proto` file and the generated `.pb.go` files in a separate directory/package and import that package (the generated Gorums API) into your application.
```

Comment:
Maybe it would be better if this guide used that approach instead?

Reply:
Opened issue #124 to address this in the future.